ports/rp2/tools_patch/Findpioasm.cmake

```diff
@@ -22,6 +22,15 @@ if (NOT TARGET pioasm)
     set(pioasmBuild_TARGET pioasmBuild)
     set(pioasm_TARGET pioasm)
 
+    # Make sure we have a version string for pioasm
```

Comment:
This is not the correct fix. The correct fix is to delete the file [Findpioasm.cmake] from tools_patch (https://github.com/micropython/micropython/blob/master/ports/rp2/tools_patch/Findpioasm.cmake)
This file was modified by Micropython to resolve an issue that should now be fixed in pico-sdk.